### PR TITLE
docs submission from feedback provided in RWG

### PR DESCRIPTION
**Pull request by vincenzospaghetti**
This is all of the feedback provided by the RWG input into Docs. Lot here - take what you want and leave the rest!

### CODE REVIEW

**Comment by vincenzospaghetti** [comment]
@jthiller i created a starting doc for open lns after feedback from RWG - can you review what I've put and add to it?

**Comment by vincenzospaghetti** [comment]
it's saying the files are empty but I'm not sure why :) @samgutentag

File: docs/oracles/oracles.mdx [hunk 1]
```diff
@@ -24,3 +24,14 @@ As such, this migration introduces several Oracles which serve as bridges betwee
...
+
+With the deprecation of the Helium L1, changes to [API] (https://docs.helium.com/api/) and [ETL] (https://github.com/helium/blockchain-etl) include:
+1. Existing ETL codebase: will no longer sync blocks.
+2. Existing API codebase: will no longer serve new data.
+3. api.helium.io (built on top of existing etl and api): will stay serving old data.
+4. explorer.helium.com (built on top of api.helium.io and other apis): will have data from the old chain and be migrated to old-explorer.helium.com.
+
+What is new includes:
+5. New datasets are available and will continue to be available on [AWS S3] (https://docs.helium.com/oracles/oracle-data)
+6. Sample API code has been published [here] ().
```
Comment (mbthiery) [inline docs/oracles/oracles.mdx]:
```suggestion
6. Sample API code has been published [here] (https://github.com/dewi-alliance/oracle-observer-sample).
```

Comment (vincenzospaghetti) [reply docs/oracles/oracles.mdx]: done - thank you!

**Review by samgutentag — APPROVED** [review]

**Pull request merged**